src/commcare_cloud/github.py

```diff
@@ -0,0 +1,60 @@
+import os
+from getpass import getpass
+from pathlib import Path
+
+from github import Github
+
+from commcare_cloud.colors import color_warning, color_notice
+
+PROJECT_ROOT = Path(__file__).parent
+GITHUB_TOKEN = None
+
+
+def github_repo(repo_name, repo_is_private=False, require_write_permissions=False):
+    token = None
+    if repo_is_private or require_write_permissions:
+        token = get_github_credentials(repo_name, repo_is_private, require_write_permissions)
+
+    repo = Github(login_or_token=token).get_repo(repo_name)
+    if require_write_permissions and not repo.permissions.push:
+        raise Exception(f"Github token does not have write permissions for '{repo_name}'")
+    return repo
+
+
+def get_github_credentials(repo_name, repo_is_private, require_write_permissions):
+    global GITHUB_TOKEN
+
+    if GITHUB_TOKEN is None:
+        GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
+
+    if GITHUB_TOKEN:
+        return GITHUB_TOKEN
+
+    try:
+        from .config import GITHUB_APIKEY
+    except ImportError:
+        try:
+            from .fab.config import GITHUB_APIKEY
+        except ImportError:
+            pass
+        else:
+            print(color_notice(f"[Deprecation Warning] Config file has moved."))
+            print(color_notice(f"New location is {PROJECT_ROOT}/config.py or else use the "
+                               f"'GITHUB_TOKEN' environment variable."))
```

Comment:
Can you add in a note about where the config file currently is or construct a `mv` command to fix it?  I imagine most people have forgotten where the config file lives.

Reply:
470b8043e